packages/components/src/components/SquiggleChart.tsx

```diff
@@ -138,7 +143,8 @@ const SquiggleItem: React.FC<SquiggleItemProps> = ({
     case "symbol":
       return (
         <VariableBox heading="Symbol" showTypes={showTypes}>
-          {expression.value}
+          <span className="text-slate-500 mr-2">Undefined Symbol:</span>
```

Comment:
Good workaround, I'd imagine this would be good to fix on the interpreter side as well